px-map-control-zoom.html

```diff
@@ -8,6 +8,9 @@
 <!-- Local dependencies -->
 <link rel="import" href="px-map-behavior-control.html">
 
+<link rel="import" href="../px-icon-set/px-icon-set.html"/>
```

Comment:
do we actually need that import here? the behavior only seem to use px-icon as a string rather than the actual element. I'm definitely confused as how this works though ;)